### DISCUSSION TITLE
Sync game cursors

### DISCUSSION THREAD

**Discussion by 0xdeadbeer** [ppy/osu#16310] [Ideas]
Since we're having multiple cursors (in specific two as I know inside osu! lazer), each with a different size I was thinking about implementing an option inside osu! settings that will sync the game cursor's size with the menu cursor's size.

Tell me your opinions please

**Comment by bdach** [comment]
...how would that work? the cursors serve different purposes and are fundamentally incomparable in size given their shape.

**Reply by 0xdeadbeer** [reply]
ughh that's true as well, although I personally did bother a bit to have different sized cursors so I thought that it might be a feature a few people might like
